src/config/GSettingsEngine.vala

```diff
@@ -137,7 +137,7 @@ public class GSettingsConfigurationEngine : ConfigurationEngine, GLib.Object {
             cleaned_id = "default";
         cleaned_id = cleaned_id.replace (".", "-");
 
-        return "org.pantheon.photos.%s.%s".printf (domain, cleaned_id);
+        return "io.elementary.photos.%s.%s".printf (domain, cleaned_id);
```

Comment:
Is this right? This looks like a gsettings thing which we'd want to not touch :grimacing:

Reply:
changed!